tests/e2e/data/config-taquito-test-environment-invalid-config-network-url.json

```diff
@@ -0,0 +1,83 @@
+{
+    "language": "en",
+    "plugins": [
+        {
+            "name": "@taqueria/plugin-taquito",
+            "type": "npm"
+        }
+    ],
+    "contractsDir": "contracts",
+    "testsDir": "tests",
+    "artifactsDir": "artifacts",
+    "environment": {
+        "default": "development",
+        "development": {
+            "networks": [],
+            "sandboxes": [
+                "local"
+            ],
+            "storage": {}
+        },
+        "test": {
+            "networks": ["hangzhounet"],
+            "sandboxes": [],
+            "storage": {"hello-tacos.tz": "100"}
+        }
+    },
+    "sandbox": {
+        "local": {
+            "accounts": {
+                "default": "bob",
+                "bob": {
+                    "initialBalance": "3000000000"
+                },
+                "alice": {
+                    "initialBalance": "2000000000"
+                },
+                "john": {
+                    "initialBalance": "4000000000"
+                },
+                "jane": {
+                    "initialBalance": "5000000000"
+                },
+                "joe": {
+                    "initialBalance": "1000000000"
+                }
+            },
+            "label": "Local Tezos Sandbox",
+            "protocol": "PtHangz2aRngywmSRGGvrcTyMbbdpWdpFKuS4uMWxg2RaH9i1qx",
+            "rpcUrl": "http://localhost:20000"
+        }
+    },
+    "network": {
+        "hangzhounet": {
+            "label": "hangzhounet",
+            "rpcUrl": "https://invalid.test",
+            "protocol": "PtHangz2aRngywmSRGGvrcTyMbbdpWdpFKuS4uMWxg2RaH9i1qx",
+            "faucet": {
```

Comment:
Ivan, one thing we have to consider - eventually after running these tests enough times, our account balances will be depleted. 

Do you mind creating an issue to look into this and add the link as a todo comment here please?

I believe we can use the keygen service to get faucets programmatically which should resolve this:
https://docs.google.com/presentation/d/1Oe6ZOE7_mX8bDh-qNKLBrsNFJzOOiELbwbz5AzOgtYs/edit#slide=id.gf347d25493_0_0

Reply:
Added new issue to address it as feature request - https://github.com/ecadlabs/taqueria/issues/378